spog/ui/crates/components/src/cyclonedx/mod.rs

```diff
@@ -15,7 +15,7 @@ pub fn cyclonedx_meta(bom: &cyclonedx_bom::prelude::Bom) -> Html {
         .metadata
         .as_ref()
         .and_then(|m| m.component.as_ref())
-        .map(|c| c.version.to_string());
+        .map(|c| c.version.as_ref().map(|e| e.to_string()).unwrap_or("".to_string()));
```

Comment:
Works! But I think it would also be possible to use `.unwrap_or_default()` in this case.

Reply:
Done! Thanks for pointing that out